### DISCUSSION TITLE
Less steps to change audio output

### DISCUSSION THREAD

**Discussion by e1a6** [File-New-Project/EarTrumpet#864] [Ideas]
It would be nice if the menu displaying the different outputs availaible for each app appeared directly after right clicking on said app instead of having to right click to make the menu with two arrows icon appear, then left click on the two arrows icon to make  the menu displaying the audio outputs appear.

**Comment by Nomad-0** [comment]
agree with this one. Or maybe it could be when I click, this menu will appear and if I right-click on an app icon, it will switch devices without the need for any other steps. Or maybe I can drag an icon from one device to another when there are more than two devices.

![image](https://user-images.githubusercontent.com/76395594/129389566-c1472541-6ffb-4c71-be59-8beda01a4ac5.png)

**Reply by theiwaz** [reply]
See #662
